### DISCUSSION TITLE
could we use this package to rename all files by elegent and professional way

### DISCUSSION THREAD

**Discussion by Adamkaram** [spacedriveapp/spacedrive#900] [Feature Request]
Szyszka is fast and powerful file renamer that make it easy to rename files with an easy way all files i mean here in some dir 
by using regex for example 
https://github.com/qarmin/szyszka

**Comment by utkubakir** [comment]
I don't see how relevant this is unless you ask if Spacedrive will support something similar. In that case yes, we will have renaming tools/extensions in the future.
